assets/js/scripts.js

```diff
@@ -388,7 +388,15 @@ $('#open-delete-all-settings-modal').on('click', function () {
   $('#delete-all-settings-modal').modal();
 });
 
-
+function timeRange(from, to){
+    // Add all valid hours to the marker to be able to simply `.includes()` it later.
+    // Could also check `if X between start and end`, might be slightly better. ¯\_(ツ)_/¯
+    var times = [];
+    for (let index = from; index !== to; (index !== 23) ? index++ : index = 0)
+        times.push(index);
+      
+      return times;
+}

```

Comment:
i think that will be more intuitive:
```javascript
function timeRange(from, to) {
  const times = [];

  let hour = from;
  while (hour !== to) {
    times.push(hour);
    hour = (hour + 1) % 24;
  }

  return times;
}
```

Reply:
I just copied what did exist. We could check if (from > to)

Reply:
no, because `if (from > to)` is always false if end time is not at the next day.
also `if (from < to)` is incorrect because if start is at 21:00 and end at 6:00 you will get only `[21, 22, 23]`.
The only correct way is (from !== to)